.github/scripts/pr_body_check.py

```
@@ -0,0 +1,64 @@
+import os
+import sys
+import re
+
+def warn(msg):
+    print(f"::warning::{msg}")
+
+def normalize(s):
+    return s.replace('\r\n', '\n').replace('\r', '\n').strip()
+
+def extract_sections(text):
```

Ruff (0.12.2)

Missing return type annotation for public function `extract_sections`

(ANN201)